react/features/recording/components/LiveStream/AbstractStartLiveStreamDialog.ts

```diff
@@ -238,14 +273,16 @@ export default class AbstractStartLiveStreamDialog<P extends IProps>
  *     _conference: Object,
  *     _googleAPIState: number,
  *     _googleProfileEmail: string,
- *     _streamKey: string
+ *     _streamKey: string,
+ *     _streamBaseUrl: string
  * }}
  */
 export function _mapStateToProps(state: IReduxState) {
     return {
         _conference: state['features/base/conference'].conference,
         _googleAPIState: state['features/google-api'].googleAPIState,
         _googleProfileEmail: state['features/google-api'].profileEmail,
-        _streamKey: state['features/recording'].streamKey
+        _streamKey: state['features/recording'].streamKey,
+        _streamBaseURL: state['features/recording'].streamBaseURL
```

Comment:
this is probably always undefined since the `streamBaseURL` is never saved to store. For native do we want to call your new action which is currently not used to save it? Or not use the store at all for `streamBaseURL`

Reply:
I guess we could use the store for streamBaseURL as we do for streamKey. I modified it like this.